src/genie/libs/parser/iosxe/tests/ShowCts/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,264 @@
+        "disconnecting": 0,
+        "invalid": 0
+    },
+    "cts_events_statistics": {
+        "authentication_success": 0,
+        "authentication_reject": 0,
+        "authentication_failure": 0,
+        "authentication_logoff": 0,
+        "authentication_no_resp": 0,
+        "authorization_success": 0,
+        "authorization_failure": 0,
+        "sap_success": 0,
+        "sap_failure": 0,
+        "port_auth_fail": 0
+    },
+    "installed_list": {
+        "name": "CTSServerList1-0089",
+        "count": 7,
+        "server_ip": {
+            "10.100.123.1": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.2": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.3": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.4": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.5": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.6": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            },
+            "10.100.123.7": {
+                "port": 1812,
+                "a_id": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                "status": "ALIVE",
+                "auto_test": "FALSE",
+                "keywrap_enable": "FALSE",
+                "idle_time": "60 mins",
+                "deadtime": "20 secs"
+            }
+        }
+    },
+    "pac_valid_until": "19:56:32 PDT Sep 6 2020",
```

Comment:
I understand just one value, but output has section with `====`, so better to have level like below:
```
'pac_summary': {
    pac_info: {
        'pac_valid_until': "19:56:32 PDT Sep 6 2020"
```
device output for this:
```
===================
CTS PAC Summary
=====================
  PAC-Info:
    PAC Valid Until: 19:56:32 PDT Sep 6 2020

Reply:
Updated